mixer/test/client/quota_cache/quota_cache_test.go

```diff
@@ -62,7 +62,11 @@ func TestQuotaCache(t *testing.T) {
 			reject++
 		}
 	}
-	if ok != 10 || reject != 10 {
+	if ok+reject < 20 {
+		t.Fatalf("sum of ok count %v and reject count %v is less than 20", ok, reject)
+	}
+	// ok should be around 10, allow 30% margin (prefetch code may have some margin).
```

Comment:
add a link to https://github.com/istio/mixerclient/tree/master/prefetch.

Reply:
@ozevren the prefetch algorithm is affected by fetch latency between client and server.
The prefetch tests https://github.com/istio/mixerclient/blob/master/prefetch/quota_prefetch_test.cc also have margins.
For `1s` window the margins are 30%.